templates/codelists/version.html

```diff
@@ -344,6 +344,8 @@ <h5 class="modal-title" id="publish-version-label">Publish Version</h5>
 </script>
 
 {{ child_map|json_script:"child-map" }}
+{{ code_to_url|json_script:"codeToURL" }}
+{{ codes_in_definition|json_script:"definitionCodes" }}
```

Comment:
Can this be `codesInDefinition`?

Reply:
The JS variable?

Reply:
Yeah -- that would make it match the pattern in all the other `json_script`s.

Reply:
Fixed in 59f3365